doc/news/_preparation_next_release.md

```diff
@@ -67,11 +67,43 @@ You can also read the [new low-level docs](../dev/kdb-operations.md) to find out
 The structure of `system:/elektra/mountpoints` changed as well.
 Take a look at the [new docs](../dev/mountpoints.md), if you need to know details.
 
+#### Updating config
+
+> **Warning**: BACK UP YOUR CONFIG FILES BEFORE UPDATING!
```

Comment:
Can you show here what should be backed up (or make the Python tool back it up).

Reply:
In theory, you don't need to back up everything. The script doesn't even overwrite anything unless you explicitly tell it to.

I practice, if you use the `--output` option, you may want to back up the file you specify before overwriting it. But that should be pretty obvious. 

I just put the warning here to be on the safe side.